integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/automaticindexing/session/AutomaticIndexingSynchronizationStrategyIT.java

```diff
@@ -45,12 +47,15 @@
 
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
 import org.apache.logging.log4j.Level;
 import org.assertj.core.api.InstanceOfAssertFactories;
 import org.awaitility.Awaitility;
 import org.hamcrest.CoreMatchers;
 
+@RunWith(Parameterized.class)
```

Comment:
Wouldn't it make more sense to just duplicate the method `success_custom` to test both submitters? Otherwise you're going to run every single method twice even though only a few of them are affected by your test parameter.

I wouldn't bother duplicating the other test methods, btw.